kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -395,107 +395,126 @@
 </script>
 
 
-<style lang="stylus" scoped>
-
-  @require '~kolibri.styles.definitions'
-
-  .fh
-    height: 100%
-
-  .wrapper-table
-    text-align: center
-    width: 100%
-    height: 100%
-    display: table
-
-  .main-row
-    display: table-row
-    background-color: $core-action-normal
-    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(./background.jpg)
-    background-repeat: no-repeat
-    background-size: cover
-    background-position: center
-    text-align: center
-
-  .main-cell
-    display: table-cell
-    vertical-align: middle
-    height: 100%
-
-  .box
-    width: 300px
-    background-color: $core-bg-light
-    margin: 16px auto
-    box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2),
-                0 1px 1px 0 rgba(0, 0, 0, 0.14),
-                0 2px 1px -1px rgba(0, 0, 0, 0.12)
-
-  .logo
-    margin-top: 16px
-
-  .login-form
-    width: 70%
-    max-width: 300px
-    position: relative
-    text-align: left
-    margin: auto
-
-  .login-btn
-    width: calc(100% - 16px)
-
-  .version
-    font-size: 0.8em
-    margin-top: 24px
-    margin-bottom: 0
-    padding-bottom: 16px
-
-  .footer-row
-    display: table-row
-    background-color: $core-bg-light
-
-  .footer-cell
-    display: table-cell
-    vertical-align: middle
-    min-height: 56px
-    padding: 16px
-
-  .suggestions
-    background-color: white
-    box-shadow: 1px 2px 8px darken(white, 10%)
-    list-style-type: none
-    margin: 0
-    width: 100%
-    padding: 0
-    z-index: 8
+<style lang="scss" scoped>
+
+  @import '~kolibri.styles.definitions';
+
+  .fh {
+    height: 100%;
+  }
+
+  .wrapper-table {
+    text-align: center;
+    width: 100%;
+    height: 100%;
+    display: table;
+  }
+
+  .main-row {
+    display: table-row;
+    background-color: $core-action-normal;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(./background.jpg);
```

Comment:
this line is giving me

```
>     /Users/d/Projects/learning_equality/repos/k011x/kolibri/plugins/user/assets/src/views/sign-in-page/index.vue
      416:1  error  Line 416 exceeds the maximum line length of 100  max-len
    
    ✖ 1 problem (1 error, 0 warnings)
```

Any chance this is happening due to `indentAndAddNewLines`?

Reply:
I think this is a prettier scss bug. This line below for example, has 102 chars but the print-width is set to 100.

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEATAlgN2AHSgAnwCMBDMAawHMAnCAVylQFp0BbEyuJfAG3SjglqTGiQwIYACmqVSkgAwAafEpXL5AOgDsASmUy5qo+u069+OtR6SNAelIUa9cuUYaAVgAdKOgNx4AXxBFEAhPGHRoAGdkUCFaAHcABSEEGJQSTAh0VGCQImoycjgYAGVPMn5KZBhqOjgQ-ii4ahgkwsp2ZAAzEh5mkPcogA8AIUKKEtKSVjgAGX44Hr6BkArqZupkECiwKJiQz2p+GAB1HJgAC2QARnklNdpm08LPbaO4TcwlkOo4AEc6Og-u0OF0kL1+g0dnBWOganVoVEqjw4ABFOgQeDLKEhGAkIjnVBXZAAJjxhXQfCglAAwhBWOCQFBoD8QHRmgAVAnpSHNAIBIA